src-ui/src/app/components/common/edit-dialog/storage-path-edit-dialog/storage-path-edit-dialog.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core'
+import { FormControl, FormGroup } from '@angular/forms'
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap'
+import { EditDialogComponent } from 'src/app/components/common/edit-dialog/edit-dialog.component'
+import { PaperlessStoragePath } from 'src/app/data/paperless-storage-path'
+import { StoragePathService } from 'src/app/services/rest/storage-path.service'
+import { ToastService } from 'src/app/services/toast.service'
+
+@Component({
+  selector: 'app-storage-path-edit-dialog',
+  templateUrl: './storage-path-edit-dialog.component.html',
+  styleUrls: ['./storage-path-edit-dialog.component.scss'],
+})
+export class StoragePathEditDialogComponent extends EditDialogComponent<PaperlessStoragePath> {
```

Comment:
This needs validation. We cannot let a user insert some crazy thing for storage path and let it break the app. It currently allows me to put anything. This probably should be done in the backend so trying to save an invalid path in some way returns an error from the API.

To be clear, validation wouldn't go here in this class, just mentioning here.

Reply:
From a security perspective, no change is required. This would be pure usability and validation. We could solve a first shot same as above.

Reply:
Sorry not sure I understand. I do mean from a usability perspective, when possible the UI tries to prevent changing things to nonsensical values. For example changing a tag color to a non hex value, entering incorrect dates, etc. I would see this as the same. The UI should provide "guardrails".

Reply:
Of course :-)

My point was: The backend already takes care of not being able to break the file storage which whatever you configure.